packages/api/src/api/helix/chat/HelixChatApi.ts

```diff
@@ -125,6 +128,36 @@ export type HelixChatUserColor =
  */
 @rtfm('api', 'HelixChatApi')
 export class HelixChatApi extends BaseApi {
+	/**
+	 * Gets the list of users that are connected to the broadcaster’s chat session.
+	 *
+	 * @beta
+	 * @param broadcasterId The ID of the broadcaster in whose list of chatters you want to get.
+	 * @param moderatorId The ID of the broadcaster or one of the broadcaster’s moderators.
```

Comment:
"The ID of" is wrong here, as you can always also pass user objects. Also I think there's a stray "in" in the first one.

Reply:
"The ID of" is used in quite a few other places (see Helix moderation API), but I updated the verbiage and parameter names here.